README.md

```diff
@@ -383,9 +383,9 @@ Calling `getItem(...)` or `...` returns an `Item` object with the following prop
   - .getMetadata(namespace) ⇒ `object|null`
   - .replaceMetadata(namespace, value, configuration) ⇒ `object`
   - .removeMetadata(namespace) ⇒ `object|null`
-  - .sendCommand(value): `value` can be a string or a [`time.ZonedDateTime`](#time)
-  - .sendCommandIfDifferent(value) ⇒ `boolean`: `value` can be a string or a [`time.ZonedDateTime`](#time)
-  - .postUpdate(value): `value` can be a string or a [`time.ZonedDateTime`](#time)
+  - .sendCommand(value): `value` can be a string, a [`time.ZonedDateTime`](#time) or a [`Quantity`](#quantity)
```

Comment:
Does value need to be a string ?  Or do we take the string representation of the value (and do proper conversion of ZonedDateTime & Quantity)?

Reply:
We can take number, string and openHAB types.
openhab-js types like Quantity or JS-Joda ZonedDateTime need to converted.
However I am not sure what you mean with your question, sorry.

Reply:
Nevermind ;-) I was not sure if when finally calling the send event method in the @runtime package  if the value is eventually converted into a string if it does not match a standard type (like a number, or something the addon specifically registers to convert  )